tests/handlers/external_test.py

```diff
@@ -2,7 +2,7 @@
 
 from __future__ import annotations
 
-from datetime import datetime, timezone
+from datetime import UTC, datetime
```

Comment:
Oh thank goodness we're finally on a new enough Python that we don't have to be sneaky about this and UTC is a top-level item in the import.